public-api/db-cdr/generate-cdr/generate_criteria_stratum.sh

```diff
@@ -44,166 +44,6 @@ then
   exit 1
 fi
 
-echo "creating criteria_stratum table rows for snomed measurements"
```

Comment:
Removed the queries that were generating rolled up counts of each stratum for snomed measurements.